core/dslmcode/shared/drupal-7.x/modules/elmsln_contrib/webcomponents/modules/webcomponents_cmstags/webcomponents_cmstags.module

```diff
@@ -318,16 +323,6 @@ function _webcomponents_cmstags_tips($filter, $format, $long = FALSE) {
  * Callback function to perform the content processing.
  */
 function _webcomponents_cmstags_process($text, $filter, $format, $langcode, $cache, $cache_id) {
-  // ensure we only apply this once based on the input format
```

Comment:
why is this block removed?

Reply:
So, this was removed as it seems to override whatever changes are made to the `window.cmstokenEndPoint` in the `webcomponents_cmstokenendpoint` hook.

A dev could use the `hook_webcomponents_cmstokenendpoint_alter` hook to update the cmstokenEndPoint value to something else. However, this block would just set it back to the default regardless.

I did just try removing this block entirely and it started using the correct endpoint I'd set in the hook and I haven't seen any issues crop up because of it (at least on our setup).